app/api/v1/endpoints/spotlight_router.py

```
@@ -0,0 +1,23 @@
+from app.schemas.spolight_dto import SpotlightDto
+from app.services.spotlight_service import SpotlightService
+
+
+class SpotlightRouter:
+
+    def __init__(self):
+        self.router = APIRouter()
+
+        self.router.add_api_route("/{target_date}", self.get_spotlight, methods=["GET"])
+
+    def get_spotlight(self, target_date: str, db: Session = Depends(get_db)) -> SpotlightDto.GetSpotlightRespDto:
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)